docs/userguides/managing.md

```diff
@@ -0,0 +1,53 @@
+# Managing Your Platform
+
+In this guide, we are going to show you more details on how to manage your [Silverback Platform](https://silverback.apeworx.io).
+
+## Managing Workspaces 
```

Comment:
Add something on how it's recommended to first signup for Silverback at silverback.apeworx.io